### ISSUE TITLE
​Show context in Bi-Directional Links block

### ISSUE THREAD

**Issue by cp7553479** [toeverything/AFFiNE#7918]
### Description

It will be very nice improvement that the Bi-Directional Links block can show the context of text indents.
Like what Logseq does
<img width="847" alt="image" src="https://github.com/user-attachments/assets/4c18487f-d2f0-4eca-bbc6-7d3b0b4ce5b6">
Showing the context of text indents helps user view contexts which belong to the link-block. It's very useful. 

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by forehalo** [comment]
@hwangdev97

**Comment by hwangdev97** [comment]
Yes, this is a bug caused by the refactoring. Originally, at least one block of content would be displayed in bi-directional links. We will fix it, but to display more content, we need further optimization. cc @EYHN

**Comment by JimmFly** [comment]
Currently, the backlinks in Bi-Directional Links can see a preview of the links. Please update to the latest version and experience it.
